public/css/main.css

```diff
@@ -120,6 +120,109 @@ body {
 
 /*** ADD EVENT FORM ***/
 
+.add-event-form {
+  max-width: 100vw;
+  padding: 3rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.add-event-form label {
+  font: bold 1.8rem 'Lato';
+  color: #5C5C5C;
+}
+
+
+.add-event-form input, select {
+  height: 3rem;
+  width: 100%;
+  border: 0.1rem gray solid;
+  border-radius: 0.3rem;
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+  background-color: white;
+  padding: 0.5rem;
+  font: 1.5rem 'Lato';
+  background-color: white;
+}
+
+.event-description {
+  padding-left: 0.5rem;
+  padding-top: 1rem;
+  height: 5rem;
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+  border-radius: 0.2rem;
+  border: 0.1rem gray solid;
+  font: 1.5rem 'Lato';
+}
+
+.checkbox-wrap {
+  padding: 1rem;
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+  border: 0.1rem gray solid;
+  flex-flow: wrap;
+  display: flex;
+  border-radius: 0.2rem;
+  background-color: #E8E8E8;
+}
+
+.checkbox-wrap label {
+  width: 50%;
+  font: bold 1.4rem 'Lato';
+  margin-top: 0.2rem;
+  margin-bottom: 0.2rem;
+  color: #5C5C5C;
+}
+
+.checkbox-wrap input {
+  height: initial;
+  width: 15%;
+}
+
+.place-section {
+  flex-direction: column;
+  display: flex;
+  width: 100%;
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+}
+
+.place-wrap {
+  margin-top: 0.5rem;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.add-place {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
```

Comment:
could you add a margin-top to this so that there's some space between the 'place' and 'name' labels when the add place inputs are open